public/utils/contextMenu/getActions.tsx

```diff
@@ -31,10 +35,11 @@ export const getActions = () => {
         toMountPoint(
           <Provider store={store}>
             <AnywhereParentFlyout
-                startingFlyout={startingFlyout}
-                embeddable={embeddable}
-                closeFlyout={() => overlay.close()}
-              />
+              startingFlyout={startingFlyout}
+              embeddable={embeddable}
+              notification={getNotifications()}
```

Comment:
You can remove this as a prop and just access notifications when you need it in the child components via `getNotifications()`

Reply:
Essentially we should never be passing services down in props - they can all just be accessed whenever we need them directly

Reply:
removed `notification={getNotifications()}` and passing it as a prop. Will do `getNotifications()` when we need to use notification service